internal/sourcemap/lineinfo.go

```diff
@@ -15,6 +15,9 @@ func CreateECMALineInfo(text string, lineStarts core.ECMALineStarts) *ECMALineIn
 }
 
 func (li *ECMALineInfo) LineCount() int {
+	if li == nil {
```

Comment:
When would this happen? It seems like we wouldn't want this to happen...

Reply:
this happens when file does not exist

Reply:
Updated TryGetSourceMappingURL to handle it instead if thats better?